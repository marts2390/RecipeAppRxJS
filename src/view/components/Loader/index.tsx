import React from 'react'

import styles from './Loader.module.scss'

const LoaderComponent = () => (
  <div className={ styles.spinner }>
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>
)

export default LoaderComponent
