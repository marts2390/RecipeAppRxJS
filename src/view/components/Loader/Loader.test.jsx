import React from 'react'
import { shallow } from 'enzyme'
import Loader from './index'

test('Slider is rendered', () => {
  const wrapper = shallow(<Loader />)
  expect(wrapper).toMatchSnapshot()
})
