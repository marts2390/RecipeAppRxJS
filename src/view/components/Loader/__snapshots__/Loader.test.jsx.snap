// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slider is rendered 1`] = `
<div
  className="spinner"
>
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
</div>
`;
